opendcs-rest-api/src/main/java/org/opendcs/odcsapi/beans/ApiTimeSeriesSpec.java

```diff
@@ -17,39 +17,69 @@
 
 import java.util.Date;
 
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "Represents the specification of a time series, including metadata and configuration details.")
 public final class ApiTimeSeriesSpec
 {
+	@Schema(description = "The time series identifier (TSID) containing unique metadata for the time series.")
 	private ApiTimeSeriesIdentifier tsid = null;
-	
+
+	@Schema(description = "The location associated with the time series.", example = "MROI4")
 	private String location = null;
+	@Schema(description = "The parameter measured by the time series (e.g., temperature, flow rate).", example = "Stage")
 	private String param = null;
+	@Schema(description = "Statistical code representing the type of calculation applied to the time series.", example = "Inst")
 	private String statCode = null;
+	@Schema(description = "The interval of time between data points in the time series.", example = "30Minutes")
 	private String interval = null;
+	@Schema(description = "The duration of time that each time series value represents.", example = "0")
 	private String duration = null;
+	@Schema(description = "Version identifier for the time series.", example = "raw")
 	private String version = null;
-	
+
+	@Schema(description = "The unique numeric identifier for the site associated with the time series.", example = "6")
```

Comment:
Same as the timeseries, but these should actually just the data. Or a least a URL instead of the ID.... though we're already doing a lot of gets on ID aren't we?

Reply:
Arguably they should at least be DbKey instead of Long. There are some open issues/discussions considering moving those to uuid: https://github.com/opendcs/opendcs/issues/625

Reply:
We'll want to do that throughout, but I wouldn't for this specific effort.